app/src/main/java/com/lagradost/quicknovel/providers/EfremnetProvider.kt

```diff
@@ -0,0 +1,1502 @@
+        Pair("Family Life", "family-life"),
+        Pair("Family & Parenting", "family-parenting"),
+        Pair("Fairytale", "fairytale"),
+        Pair("Exploitation", "exploitation"),
+        Pair("Existentialism", "existentialism"),
+        Pair("Ethiopia", "ethiopia"),
+        Pair("Essays & Memoirs", "essays-memoirs"),
+        Pair("Emotion", "emotion"),
+        Pair("Educational", "educational"),
+        Pair("Eco Fiction", "eco-fiction"),
+        Pair("Dieselpunk", "dieselpunk"),
+        Pair("Devotion and Inspiration", "devotion-and-inspiration"),
+        Pair("Development", "development"),
+        Pair("Dark Suspense Erotica", "dark-suspense-erotica"),
+        Pair("Dark Scifi / Fantasy", "dark-scifi-fantasy"),
+        Pair("Dark Fantasy Romance", "dark-fantasy-romance"),
+        Pair("Dark Erotica Romance", "dark-erotica-romance"),
+        Pair("Dark Dominant Men", "dark-dominant-men"),
+        Pair("Danish", "danish"),
+        Pair("Cthulhu Mythos", "cthulhu-mythos"),
+        Pair("Creationism", "creationism"),
+        Pair("Counting", "counting"),
+        Pair("Cornwall", "cornwall"),
+        Pair("Comtemporary", "comtemporary"),
+        Pair("Comedy/humor", "comedy-humor"),
+        Pair("Colouring", "colouring"),
+        Pair("Collapse", "collapse"),
+        Pair("Collage", "collage"),
+        Pair("Classic Literature", "classic-literature"),
+        Pair("Cinema & Theater", "cinema-theater"),
+        Pair("Children's Books and Young Adult", "children-s-books-and-young-adult"),
+        Pair("Central Africa", "central-africa"),
+        Pair("Catastrophism", "catastrophism"),
+        Pair("Cat Sleuths", "cat-sleuths"),
+        Pair("Cars", "cars"),
+        Pair("Career", "career"),
+        Pair("Canadian", "canadian"),
+        Pair("Bulgaria", "bulgaria"),
+        Pair("Book Club Fiction/literary Ficti", "book-club-fiction-literary-ficti"),
+        Pair("Bolivia", "bolivia"),
+        Pair("Bisexual LGBTQIA", "bisexual-lgbtqia"),
+        Pair("Biochemistry", "biochemistry"),
+        Pair("Batman", "batman"),
+        Pair("Banking", "banking"),
+        Pair("BDSM Erotic", "bdsm-erotic"),
+        Pair("Atlantis", "atlantis"),
+        Pair("Asian Studies", "asian-studies"),
+        Pair("Archaeology and History", "archaeology-and-history"),
+        Pair("Apocalypse", "apocalypse"),
+        Pair("Ancient", "ancient"),
+        Pair("Anarchism", "anarchism"),
+        Pair("Americana", "americana"),
+        Pair("American South", "american-south"),
+        Pair("American Revolutionary War", "american-revolutionary-war"),
+        Pair("American Libertarianism", "american-libertarianism"),
+        Pair("Amateur Sleuth", "amateur-sleuth"),
+        Pair("All Ages", "all-ages"),
+        Pair("Alexander The Great", "alexander-the-great"),
+        Pair("African Literature", "african-literature"),
+        Pair("Adventure Thriller", "adventure-thriller"),
+    )
```

Comment:
with this many providers, add `.sortedBy { it.first }` to make it sort alphabetically